test-helpers/assertions.js

```diff
@@ -22,48 +22,12 @@ exports.assertSustainabilityTarget = async (
 ) => {
   const currentTarget = (
     await instance.getMp(
-      (await instance.latestMpIds(address)).toNumber()
+      (await instance.latestMpNumber(address)).toNumber()
     )
   ).target.toNumber();
   assert.equal(currentTarget, target, message);
 };
 
-exports.assertInitializeMoneyPoolEvent = async (
```

Comment:
Not testing events anymore?

Reply:
i removed the initialize event and the activate event. Those were both events i added unsure of how the front-end would need them. Turns out we're good to go without them.